pkg/agent/node/node.go

```diff
@@ -51,6 +53,18 @@ func Params() map[string]interface{} {
 	}
 }
 
+func parseLabel(v string) map[string]string {
+	labels := map[string]string{}
+	parts := strings.Split(v, ",")
+	for _, part := range parts {
+		kvs := strings.Split(part, "=")
+		if len(kvs) == 2 {
```

Comment:
do you think warn on incorrect format would be useful? when len != 2

Reply:
@alena1108 Yeah, i can add warn. Also i don't know if `foo=` or `foo` would be valid cases.